### PR TITLE
Add Pisth by @ColdGrub1384

### PR DESCRIPTION
**Pull request by ColdGrub1384**
1. [x] Project URL: https://pisth.github.io
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Avoid iOS or open-source in description as it is assumed
5. [x] Use this commit title format if applicable: Add app-name by @github-username
6. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -9039,6 +9039,22 @@
       ]
     },
     {
+++    "title": "Pisth",
```
Comment (dkhamsing) [inline contents.json]: I think these in red are some special characters..

**Comment by dkhamsing** [comment]
@ColdGrub1384 hey dont worry about the issues, we will help you add this project

Thanks!

**Comment by ColdGrub1384** [comment]
I fixed it, thanks. I copy pasted from github.com, so there was the "++".

**Comment by ColdGrub1384** [comment]
Does it matter if there are 2 commits for one application?

**Comment by ColdGrub1384** [comment]
I also fixed the 404 error for a screenshot, but from server side

**Comment by dkhamsing** [comment]
Thx for the fixes

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request merged**